src/fontra/views/editor/editor.js

```diff
@@ -424,6 +425,27 @@ export class EditorController {
   }
 
   initSidebars() {
+    const editorContainer = document.querySelector(".editor-container");
+    const sidebars = [new Sidebar("left"), new Sidebar("right")];
+    this.sidebars = sidebars;
+
+    this.addSidebarPanel(new TextEntryPanel(), "left");
+    this.addSidebarPanel(new GlyphSearchPanel(), "left");
+    this.addSidebarPanel(new DesignspaceNavigationPanel(), "left");
+    this.addSidebarPanel(new UserSettingsPanel(), "left");
+    this.addSidebarPanel(new ReferenceFontPanel(), "left");
+    this.addSidebarPanel(new SelectionInfoPanel(), "right");
+
+    for (const sidebar of sidebars) {
+      sidebar.attach(editorContainer);
+    }
+
+    for (const sidebar of sidebars) {
```

Comment:
I think this could/should be handled by `sidebar.addPanel(...)`?

A single call to `this.addSidebarPanel(...)` should be enough to add a new sidebar panel.